editor/src/components/canvas/canvas-strategies/strategies/relative-move-strategy.tsx

```diff
@@ -30,7 +29,10 @@ export function relativeMoveStrategy(
   const filteredSelectedElements = getDragTargets(selectedElements)
   const last = filteredSelectedElements[filteredSelectedElements.length - 1]
   const metadata = MetadataUtils.findElementByElementPath(canvasState.startingMetadata, last)
-  if (metadata == null || metadata.specialSizeMeasurements.position !== 'relative') {
+  if (
+    metadata == null ||
+    metadata.specialSizeMeasurements.position !== 'relative' // I think this check should be filteredSelectedElements.every
```

Comment:
Looks like it.

Reply:
I'm going to leave this for 2 reasons:
1) This comment was made by @balazsbajorics :D
2) I don't want to change any unrelated behaviour in this PR.